pkg/controllermanager/controller/certificatesigningrequest/csr_autoapprove_control.go

```diff
@@ -30,6 +30,8 @@ import (
 	certificatesv1beta1 "k8s.io/api/certificates/v1beta1"
 	v1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	certificatesClientV1 "k8s.io/client-go/kubernetes/typed/certificates/v1"
+	certificatesClientV1beta1 "k8s.io/client-go/kubernetes/typed/certificates/v1beta1"
```

Comment:
Do not use camelCase for import aliases. On the way, you could rename the current `v1` alias to `corev1`